### PR TITLE
Feature/apis filter

### PR DESCRIPTION
**Pull request by marla-singer**
Parts of issue #8 
### New
- Create the new collection helper `managedApiIds`, `filteredApis`
- Create re-used template for APIs filter
### Refactor
- Refactor the old collection helpers using new helper
- Refactor autorun using new helper

Depends on #255 PR. Don't merge PR until the situation hasn't become clear

### CODE REVIEW

File: api_filter/client/form/filter_form.css [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+    padding: 10px;
+    width: 30%;
+    visibility: hidden;
+    box-shadow: 0 2px 5px rgba(0,0,0,0.15);
+    opacity: 0;
+    z-index: 100;
+    transition: all .3s;
+}
+
+.filter-popup-showed {
```
Comment (brylie) [inline api_filter/client/form/filter_form.css]: Lets call this `filter-popup-visible`.

File: api_filter/client/form/filter_form.js [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+      'testing',
+    ]
+  },
+})
+Template.apisFilterForm.events({
+  'change [name=lifecycle]': (event, templateInstance) => {
+    // Get & save value of selected item
+    templateInstance.lifecycle = event.currentTarget.value;
+    if (event.currentTarget.value) {
+
```
Comment (brylie) [inline api_filter/client/form/filter_form.js]: What is this if block for?

File: api_filter/client/form/filter_form.js [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+      FlowRouter.setQueryParams({ lifecycle: tag });
+    }
+
+    $('.filter-popup').toggleClass('filter-popup-showed');
+
+  },
+  'click [type=reset]': (event, templateInstance) => {
+    // Delete tag from
+    // templateInstance.lifecycle = undefined;
+
```
Comment (brylie) [inline api_filter/client/form/filter_form.js]: Remove empty line

File: organizations/client/profile/profile.js [hunk 4]
```diff
@@ -32,50 +36,70 @@ Template.organizationProfile.onCreated(function () {
...
-      managedApisCount = organization.apisCount();
+    // Placeholder for storaging APIs of current organization
+    let organizationApis;
+
+    // Get query parameter LifeCycle
+    const lifecycleParameter = FlowRouter.getQueryParam('lifecycle');
+    // Save selected option in reactive var
+    instance.selectedFilterOption.set(lifecycleParameter);
+
+
```
Comment (brylie) [inline organizations/client/profile/profile.js]: Remove extra empty line

**Comment by mauriciovieira** [comment]
Can you rebase this branch on develop to get the latest `.eslintignore`?

> `git checkout develop`
> `git pull` (to get the latest develop) 
> `git checkout feature/apis-filter` (go back to your branch)
> `git rebase develop` (bring develop branch and put your commits on top)
> `git push --force origin/feature/apis-filter` (as you changed your tree, you will not be able to push without forcing)

After this, you will have .eslintignore with the ignored files. You should remove all files you touch in this pull requests from .eslintignore.

At this point, travis-ci will check the lint for you. You can also always do `npm run-script lint` or `meteor npm run-script lint` to check.

**Comment by marla-singer** [comment]
@brylie Fix requested changes also I linted my project

**Comment by brylie** [comment]
This code looks good. We checked it out at review meeting on Friday.

**Pull request merged**